lavamoat/browserify/beta/policy.json

```diff
@@ -1682,6 +1683,23 @@
         "punycode": true
       }
     },
+    "@metamask/assets-controllers>@metamask/phishing-controller": {
```

Comment:
peer-dependencies have been changed to be dependencies. Hence this new addition.